pkg/kubevirt/apis/provider_spec.go

```diff
@@ -16,21 +16,22 @@ package api
 
 import (
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	kubevirtv1 "kubevirt.io/client-go/api/v1"
 )
 
 // KubeVirtProviderSpec is the spec to be used while parsing the calls.
 type KubeVirtProviderSpec struct {
+	// Resources defines requests and limits resources of VMI
+	Resources *kubevirtv1.ResourceRequirements `json:"resources"`
```

Comment:
Since this is not really optional, could we use `kubevirtv1.ResourceRequirements` (not a pointer)?